logo.md

```diff
@@ -1,3 +1,3 @@
-The logo is © 2010-2014 Sensio Labs.
+The logo is © 2010-2021 Sensio Labs.
```

Comment:
Can we drop an ending year? I've read that single year as starting year of copyright is good too.

Or, can we go with 2030, as apparently no one care?

The funniest thing here is that the file was added to repo last year 😆

Reply:
I don't know and I don't have the time to check, lets leave it for now than